src/App.jsx

```diff
@@ -2,15 +2,14 @@ import { Routes, Route } from "react-router-dom";
 
 import { Home } from "./Pages";
 
-import { Navbar } from "./Components";
+import { Navbar,Button,Footer } from "./Components";
+
 
 const App = () => {
   return (
     <>
-      <Navbar />
-      <Routes>
-        <Route path="/" element={<Home />} />
-      </Routes>
+      <Footer/>
+     
```

Comment:
Why have you removed the Navbar and Routes??? They are for navigation in your page. Add them back and put the Footer below the closing </Routes> tag.